### DISCUSSION TITLE
Create a tool that easier the use of notion_sdk_py

### DISCUSSION THREAD

**Discussion by bluewhitep** [ramnes/notion-sdk-py#175] [Show and tell]
Hello.
I think Notion_sdk_py Usage documentation is less, cumbersome to use, and too many dictionary operations. 
And I Create a tool for easier the use of notion_sdk_py.

You are very welcome to try it out!
Repositories: https://github.com/bluewhitep/Notion_kit

**Comment by ramnes** [comment]
Did you see [notional](https://github.com/jheddings)? It has the same goals, a nice API, and comrade @jheddings put quite some work on it already. :)

**Reply by bluewhitep** [reply]
Yes, I saw that.
It's a very nice api, thanks a lot for the work @jheddings.
But it's not that simple for my personal abilities. I needed a more straightforward way to use it, so I made notion_kit in the hope that it would be an option to use notion easily. :)
I‘m sorry, my English is not that good.

**Reply by jheddings** [reply]
@bluewhitep - Nice work on the simple interface and making the connection!

I'd be happy to help with more [examples](https://github.com/jheddings/notional/tree/main/examples) or adding specific support to Notional if there is a use case that would benefit others.  Drop by and [start a discussion](https://github.com/jheddings/notional/discussions) if you are interested!
